### ISSUE TITLE
DietPi-Software | Home Assistant  - Supervised

### ISSUE THREAD

**Issue by HyperCriSiS** [MichaIng/DietPi#4709]
Hi,

is there any chance that you can add a supervised Home Assistant installation?
https://github.com/home-assistant/supervised-installer

You have much more possibilities with a supervised version :)

Regards,
Hyper

**Comment by Joulinar** [comment]
Hi,

honestly I don't think so. Best to my knowledge it require `Network Manager`, which is not used on DietPi. I did some test installation in past. Just have a look into our forum https://dietpi.com/phpbb/viewtopic.php?p=29512#p29512

@MichaIng 
your view?

**Comment by HyperCriSiS** [comment]
Maybe with a warning that some things are broken then?
HA is so restricted if not supervised :(

**Comment by Joulinar** [comment]
@MichaIng 
could explain it better thy I. But if I'm not mistaken `Network Manager` is conflicting with `ifupdown` used by DietPi and it will break DietPi network configuration options.

**Comment by HyperCriSiS** [comment]
The only solution would be a docker installation then?

**Comment by Joulinar** [comment]
the installation test I did ended up in a mixture of docker container and components (Network Manager) outside the container. Just check the forum post I linked above.

**Comment by HyperCriSiS** [comment]
OK, thank you! I will try the installation.

**Comment by MichaIng** [comment]
This basically turns the OS into Home Assistant OS. I'm not sure about the benefit compared to flashing Home Assistant OS in the first place, it does not even support any other SBC then the ones where Home Assistant OS images are available for.

We ship Home Assistant (core) for doing what its supposed to do: Managing smart home devices. This is the only way we can guarantee compatibility with other software installed for their purpose, which is one of the main goals of DietPi. And I'm also not keen to have two different HA install options, as those would be the case: The way Home Assistant supervised is installed has nothing to do with the way Home Assistant core is installed, it is a complete different "product".

I can understand that it is convenient to have one web interface to manage everything, but often, the more it can do, the more it demands a specific setup and limits compatibility with other software. And finally there is nothing left for having DietPi as the whole system is turned into HA OS, and we had to deal with a lot of reports why this and that is not working anymore 😉.

**Comment by MichaIng** [comment]
Btw, I just updated the FeatHub request, which still had the old name HASSIO: https://feathub.com/MichaIng/DietPi/+105

As it is a very frequently requested implementation, I'll keep this open and maybe I find time to do a test install and see whether we can make it work with ifupdown and limit or change its other demands. But as said, from all I read, even the statements of the HA team themselves, you should simply flash Home Assistant OS if you want this whole package.

**Comment by HyperCriSiS** [comment]
Thank you! It would be so nice to have all HA features available on Dietpi.

**Comment by HyperCriSiS** [comment]
> 
> 
> Hi,
> 
> honestly I don't think so. Best to my knowledge it require `Network Manager`, which is not used on DietPi. I did some test installation in past. Just have a look into our forum https://dietpi.com/phpbb/viewtopic.php?p=29512#p29512
> 
> @Michag
> your view?

The Installation worked very well. Unfortunately Pihole is broken then. No IP v4 Adpters bound and no valid ipv6 entries in setupVars.
Do you know if there is any possibility to manually fix this?

**Comment by MichaIng** [comment]
> and we had to deal with a lot of reports why this and that is not working anymore 😉.

😉

Not having Network Manager installed in the first place would be required for us to even think about implementing it. I don't know about how Network Manager sets up interfaces by default and how HA Supervised may further configure and change this away from what our tools and Pi-hole expect or are able to deal with. To get info about the apparent network setup, use the following three commands to list adapter info, assigned IP addresses and routes:
```sh
ip l
ip a
ip r
```

Would be interesting what happened if you simply purge Network Manager again (breaks network connection at first) and let the hopefully still installed ifupdown again take over (after a reboot). Major question is whether HA Supervised actively makes use of NetMan, or whether only the installer wants it to have a consistent network stack across instances, while every other network stack works just fine. Docker at least sets up the host <> guest network independently.

**Comment by HyperCriSiS** [comment]
> ```shell
> ip l
> ip a
> ip r
> ```

This is the output:
```

1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000
    link/ether dc:a6:32:2f:5c:be brd ff:ff:ff:ff:ff:ff
3: hassio: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default 
    link/ether 02:42:90:2c:1e:8b brd ff:ff:ff:ff:ff:ff
4: docker0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default 
    link/ether 02:42:db:88:8d:78 brd ff:ff:ff:ff:ff:ff
6: veth6064ee3@if5: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master hassio state UP mode DEFAULT group default 
    link/ether 96:71:9d:86:d6:f7 brd ff:ff:ff:ff:ff:ff link-netnsid 0
8: vethde90985@if7: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master docker0 state UP mode DEFAULT group default 
    link/ether 42:b0:03:a3:f8:b3 brd ff:ff:ff:ff:ff:ff link-netnsid 1
10: veth8bdbf21@if9: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master hassio state UP mode DEFAULT group default 
    link/ether 7e:dd:32:55:eb:28 brd ff:ff:ff:ff:ff:ff link-netnsid 1
12: veth7e8d1d0@if11: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master hassio state UP mode DEFAULT group default 
    link/ether ee:26:0f:80:b5:11 brd ff:ff:ff:ff:ff:ff link-netnsid 2
14: vethaf80a34@if13: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master hassio state UP mode DEFAULT group default 
    link/ether ee:21:4d:d7:37:2f brd ff:ff:ff:ff:ff:ff link-netnsid 3
16: veth5f5770a@if15: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master hassio state UP mode DEFAULT group default 
    link/ether da:66:73:93:41:58 brd ff:ff:ff:ff:ff:ff link-netnsid 4
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    link/ether dc:a6:32:2f:5c:be brd ff:ff:ff:ff:ff:ff
    inet 192.168.0.103/24 brd 192.168.0.255 scope global dynamic noprefixroute eth0
       valid_lft 1778892sec preferred_lft 1778892sec
    inet6 2003:de:d71f:8f49:ac1b:ad60:b859:9cc7/64 scope global dynamic noprefixroute 
       valid_lft 7201sec preferred_lft 1sec
    inet6 2003:de:d719:2a5d:1286:a8d7:8b04:9030/64 scope global dynamic noprefixroute 
       valid_lft 172802sec preferred_lft 86402sec
    inet6 fe80::ab28:b984:9c54:e235/64 scope link noprefixroute 
       valid_lft forever preferred_lft forever
3: hassio: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 02:42:90:2c:1e:8b brd ff:ff:ff:ff:ff:ff
    inet 172.30.32.1/23 brd 172.30.33.255 scope global hassio
       valid_lft forever preferred_lft forever
    inet6 fe80::42:90ff:fe2c:1e8b/64 scope link 
       valid_lft forever preferred_lft forever
4: docker0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether 02:42:db:88:8d:78 brd ff:ff:ff:ff:ff:ff
    inet 172.17.0.1/16 brd 172.17.255.255 scope global docker0
       valid_lft forever preferred_lft forever
    inet6 fe80::42:dbff:fe88:8d78/64 scope link 
       valid_lft forever preferred_lft forever
6: veth6064ee3@if5: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master hassio state UP group default 
    link/ether 96:71:9d:86:d6:f7 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet6 fe80::9471:9dff:fe86:d6f7/64 scope link 
       valid_lft forever preferred_lft forever
8: vethde90985@if7: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master docker0 state UP group default 
    link/ether 42:b0:03:a3:f8:b3 brd ff:ff:ff:ff:ff:ff link-netnsid 1
    inet6 fe80::40b0:3ff:fea3:f8b3/64 scope link 
       valid_lft forever preferred_lft forever
10: veth8bdbf21@if9: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master hassio state UP group default 
    link/ether 7e:dd:32:55:eb:28 brd ff:ff:ff:ff:ff:ff link-netnsid 1
    inet6 fe80::7cdd:32ff:fe55:eb28/64 scope link 
       valid_lft forever preferred_lft forever
12: veth7e8d1d0@if11: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master hassio state UP group default 
    link/ether ee:26:0f:80:b5:11 brd ff:ff:ff:ff:ff:ff link-netnsid 2
    inet6 fe80::ec26:fff:fe80:b511/64 scope link 
       valid_lft forever preferred_lft forever
14: vethaf80a34@if13: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master hassio state UP group default 
    link/ether ee:21:4d:d7:37:2f brd ff:ff:ff:ff:ff:ff link-netnsid 3
    inet6 fe80::ec21:4dff:fed7:372f/64 scope link 
       valid_lft forever preferred_lft forever
16: veth5f5770a@if15: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master hassio state UP group default 
    link/ether da:66:73:93:41:58 brd ff:ff:ff:ff:ff:ff link-netnsid 4
    inet6 fe80::d866:73ff:fe93:4158/64 scope link 
       valid_lft forever preferred_lft forever
default via 192.168.0.1 dev eth0 proto dhcp metric 100 
172.17.0.0/16 dev docker0 proto kernel scope link src 172.17.0.1 
172.30.32.0/23 dev hassio proto kernel scope link src 172.30.32.1 
192.168.0.0/24 dev eth0 proto kernel scope link src 192.168.0.103 metric 100 
```

**Comment by HyperCriSiS** [comment]
> ive used the install script on dietpi many times without issue. i always use dietpi as the base. with that said i've noticed on bullseye you have to be careful when making network changes after installing home assistant. make sure you dont say yes when asked if you want to uninstall un-needed things as it will uninstall network manager and it's dependencies and it isnt fun to fix

This sounds good! Can you maybe explain the exact steps how you successfully did it?

**Comment by MichaIng** [comment]
> default via 192.168.0.1 dev eth0 proto dhcp metric 100

One thing is to assign your IP as static IP. Not sure whether NetMan took over and changed it to DHCP, and no idea how to set it up as static IP in NetMan, if so, though. That may already be the issue with Pi-hole, when the IP changed due to DHCP.

**Comment by HyperCriSiS** [comment]
> > default via 192.168.0.1 dev eth0 proto dhcp metric 100
> 
> One thing is to assign your IP as static IP. Not sure whether NetMan took over and changed it to DHCP, and no idea how to set it up as static IP in NetMan, if so, though. That may already be the issue with Pi-hole, when the IP changed due to DHCP.

ahhh, yes... This was changed, it was static before.

**Comment by pbanj** [comment]
@HyperCriSiS I had to manually grab netman, and pretty much all it's dependencies and their dependencies. I then moved them into the cache location for apt and then used apt to install everything as it was easier than trying to do all the deb packages. 

I had reinstalled dietpi a few times because of this issue until I noticed exactly what was causing it, then I went about fixing it. After I was done I took a full image of my sdd so I can just restore to an already all set up state if I ever have to reinstall.


Also rebooting didn't help at all, and manually setting the network info in dietpi-config didn't help either.

**Comment by Joulinar** [comment]
That's exactly the point. Installing netman will break`dietpi-config`

**Comment by HyperCriSiS** [comment]
> 
> 
> @HyperCriSiS I had to manually grab netman, and pretty much all it's dependencies and their dependencies. I then moved them into the cache location for apt and then used apt to install everything as it was easier than trying to do all the deb packages.
> 
> I had reinstalled dietpi a few times because of this issue until I noticed exactly what was causing it, then I went about fixing it. After I was done I took a full image of my sdd so I can just restore to an already all set up state if I ever have to reinstall.
> 
> Also rebooting didn't help at all, and manually setting the network info in dietpi-config didn't help either.

omg, I am not so familiar with Linux so I think I can not fix this. =D
Would it be possible that you share this image, if it is without personal information?

**Comment by pbanj** [comment]
> 
> 
> > @HyperCriSiS I had to manually grab netman, and pretty much all it's dependencies and their dependencies. I then moved them into the cache location for apt and then used apt to install everything as it was easier than trying to do all the deb packages.
> > I had reinstalled dietpi a few times because of this issue until I noticed exactly what was causing it, then I went about fixing it. After I was done I took a full image of my sdd so I can just restore to an already all set up state if I ever have to reinstall.
> > Also rebooting didn't help at all, and manually setting the network info in dietpi-config didn't help either.
> 
> omg, I am not so familiar with Linux so I think I can not fix this. =D
> Would it be possible that you share this image, if it is without personal information?

it has personal stuff in it as i run a bunch more than just home assistant on this pi. but i will see if i still have all the packages saved and ill zip them up and post them

**Comment by pbanj** [comment]
ok i dont still have it. do apt install network-manager and give me the list of dependencies it say's missing. and i will grab them all and send it with instructions on how to do it @HyperCriSiS

**Comment by HyperCriSiS** [comment]
> 
> 
> > > @HyperCriSiS I had to manually grab netman, and pretty much all it's dependencies and their dependencies. I then moved them into the cache location for apt and then used apt to install everything as it was easier than trying to do all the deb packages.
> > > I had reinstalled dietpi a few times because of this issue until I noticed exactly what was causing it, then I went about fixing it. After I was done I took a full image of my sdd so I can just restore to an already all set up state if I ever have to reinstall.
> > > Also rebooting didn't help at all, and manually setting the network info in dietpi-config didn't help either.
> > 
> > 
> > omg, I am not so familiar with Linux so I think I can not fix this. =D
> > Would it be possible that you share this image, if it is without personal information?
> 
> it has personal stuff in it as i run a bunch more than just home assistant on this pi. but i will see if i still have all the packages saved and ill zip them up and post them
Awesome, thank you so much!
I wonder how the deltion is done. 
I just had a look at the installation script, but I can't find anything related to deletion of packages.
https://raw.githubusercontent.com/Kanga-Who/home-assistant/master/supervised-installer.sh

**Comment by pbanj** [comment]
> 
> 
> > it has personal stuff in it as i run a bunch more than just home assistant on this pi. but i will see if i still have all the packages saved and ill zip them up and post them
> 
> Awesome, thank you so much!
> I wonder how the deletion is done.
> I just had a look at the installation script, but I can't find anything related to deletion of packages.
> https://raw.githubusercontent.com/Kanga-Who/home-assistant/master/supervised-installer.sh

it isnt ha doing the uninstall. it is dietpi if you make changes to the network after installing ha. it will ask if you want to remove stuff you dont need anymore. saying yes is what makes it delete stuff. if you havent done too much on the system i would just grab a backup of your ha install and any other stuff you might need and reinstall dietpi as it is the easiest and fastest way to fix

**Comment by Joulinar** [comment]
again guys, if you install HA supervised, you should not use dietpi-config anymore to adjust network details. This has been taken over by Network Manager. Hense nothing will be uninstalled. This is exactly the downside of HA supervised.

**Comment by pbanj** [comment]
> 
> 
> again guys, if you install HA supervised, you should not use dietpi-config anymore to adjust network details. This has been taken over by Network Manager. Hense nothing will be uninstalled. This is exactly the downside of HA supervised.

ya you've said this. I'm simply trying to help him fix it now that it is broken, which is pretty obvious. 

As for a "fix" for dietpi maybe put a warning in the network section of dietpi-config telling people not to use it if they are using ha supervised.

**Comment by HyperCriSiS** [comment]
The best installation method seems to be a container.
[https://www.home-assistant.io/installation/linux#install-home-assistant-container](url)

Can we maybe integrate this :grinning:?

**Comment by Joulinar** [comment]
I guess you linked the HA version without the supervised stuff. Am I right?

**Comment by HyperCriSiS** [comment]
> I guess you linked the HA version without the supervised stuff. Am I right?

Yeah I think you're right. Seems not to be supervised. :(
The method that you successfully tested, can't this be integrated into the DietPi installer?

Incredible how much you guys did in the last months. The new Dashboard looks awesome.

**Comment by Joulinar** [comment]
The problem is still network manager as it is conflicting with our network setup.

**Comment by MichaIng** [comment]
Respectively outstanding tests whether the container can be made running without NetworkManager.

**Comment by Joulinar** [comment]
I would like to test but my RPI4B is blocked by a Discourse testing instance. 😉 

Maybe I will setup VM for HA Supervised

**Comment by HyperCriSiS** [comment]
😕Unfortunately my Linux skills are too low to find out how to fix this

**Comment by pbanj** [comment]
i now have a spare pi i can mess around with. im going to see if i can get this to work just by messing with the docker containers

**Comment by pbanj** [comment]
ok so i got everything but home assistant running. 
![image](https://user-images.githubusercontent.com/17306233/168513471-b682bf14-aff1-44d9-a8fc-cca191fb524d.png)
no apparmor, no network manager
it yells in the log but it still runs.
![image](https://user-images.githubusercontent.com/17306233/168513851-e613cb2b-b09e-401f-85d8-779448f07be1.png)

`[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] udev.sh: executing... 
[03:13:40] INFO: Using udev information from host
[cont-init.d] udev.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[03:13:40] INFO: Starting local supervisor watchdog...
22-05-16 03:13:46 INFO (MainThread) [__main__] Initializing Supervisor setup
22-05-16 03:13:46 INFO (MainThread) [supervisor.bootstrap] Initializing Supervisor Sentry
22-05-15 23:13:46 INFO (MainThread) [supervisor.bootstrap] Seting up coresys for machine: raspberrypi4-64
22-05-15 23:13:46 INFO (SyncWorker_0) [supervisor.docker.supervisor] Attaching to Supervisor ghcr.io/home-assistant/aarch64-hassio-supervisor with version 2022.05.2
22-05-15 23:13:46 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.INITIALIZE
22-05-15 23:13:46 WARNING (MainThread) [supervisor.resolution.evaluations.base] AppArmor is required for Home Assistant. (more-info: https://www.home-assistant.io/more-info/unsupported/apparmor)
22-05-15 23:13:46 WARNING (MainThread) [supervisor.resolution.evaluations.base] D-Bus is required for Home Assistant. (more-info: https://www.home-assistant.io/more-info/unsupported/dbus)
22-05-15 23:13:46 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-05-15 23:13:47 INFO (MainThread) [__main__] Setting up Supervisor
22-05-15 23:13:47 INFO (MainThread) [supervisor.api] Starting API on 172.30.32.2
22-05-15 23:13:47 INFO (MainThread) [supervisor.hardware.monitor] Started Supervisor hardware monitor
22-05-15 23:13:47 ERROR (MainThread) [supervisor.dbus.manager] No D-Bus support on Host. Disabled any kind of host control!
22-05-15 23:13:47 INFO (MainThread) [supervisor.host.info] Updating local host information
22-05-15 23:13:47 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
22-05-15 23:13:47 INFO (MainThread) [supervisor.host.manager] Host information reload completed
22-05-15 23:13:47 INFO (MainThread) [supervisor.jobs] 'NetworkManager.load' blocked from execution, host Network Manager not available
22-05-15 23:13:47 INFO (MainThread) [supervisor.host.apparmor] Loading AppArmor Profiles: set()
22-05-15 23:13:47 WARNING (MainThread) [supervisor.host.apparmor] AppArmor is not enabled on host
22-05-15 23:13:47 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-cli with version 2022.05.0
22-05-15 23:13:47 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-dns with version 2022.04.1
22-05-15 23:13:47 INFO (MainThread) [supervisor.plugins.dns] Updated /etc/resolv.conf
22-05-15 23:13:47 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-audio with version 2022.05.0
22-05-15 23:13:47 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-observer with version 2021.10.0
22-05-15 23:13:47 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/aarch64-hassio-multicast with version 2022.02.0
22-05-15 23:13:47 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
22-05-15 23:13:51 INFO (MainThread) [supervisor.homeassistant.secrets] Loaded 0 Home Assistant secrets
22-05-15 23:13:51 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/raspberrypi4-64-homeassistant with version 2022.5.4
22-05-15 23:13:51 INFO (MainThread) [supervisor.os.manager] No Home Assistant Operating System found
22-05-15 23:13:52 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/a0d7b954 repository
22-05-15 23:13:52 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/core repository
22-05-15 23:13:52 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/5c53de3b repository
22-05-15 23:13:53 INFO (MainThread) [supervisor.store] Loading add-ons from store: 65 all - 65 new - 0 remove
22-05-15 23:13:53 INFO (MainThread) [supervisor.addons] Found 0 installed add-ons
22-05-15 23:13:53 INFO (MainThread) [supervisor.backups.manager] Found 0 backup files
22-05-15 23:13:53 INFO (MainThread) [supervisor.discovery] Loaded 0 messages
22-05-15 23:13:53 INFO (MainThread) [supervisor.ingress] Loaded 0 ingress sessions
22-05-15 23:13:53 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.SETUP
22-05-15 23:13:53 INFO (MainThread) [supervisor.resolution.check] System checks complete
22-05-15 23:13:53 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
22-05-15 23:13:53 WARNING (MainThread) [supervisor.resolution.evaluations.base] Systemd-Resolved is required for DNS in Home Assistant. (more-info: https://www.home-assistant.io/more-info/unsupported/systemd_resolved)
22-05-15 23:13:53 WARNING (MainThread) [supervisor.resolution.evaluations.base] Systemd is not correctly working (more-info: https://www.home-assistant.io/more-info/unsupported/systemd)
22-05-15 23:13:53 WARNING (MainThread) [supervisor.resolution.evaluations.base] Detected unsupported OS: None (more-info: https://www.home-assistant.io/more-info/unsupported/os)
22-05-15 23:13:53 WARNING (MainThread) [supervisor.resolution.evaluations.base] NetworkManager is not correctly configured (more-info: https://www.home-assistant.io/more-info/unsupported/network_manager)
22-05-15 23:13:53 WARNING (MainThread) [supervisor.resolution.evaluations.base] OS-Agent is not correctly working (more-info: https://www.home-assistant.io/more-info/unsupported/os_agent)
22-05-15 23:13:53 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-05-15 23:13:53 INFO (MainThread) [supervisor.jobs] 'ResolutionFixup.run_autofix' blocked from execution, system is not running - CoreState.SETUP
22-05-15 23:13:53 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
22-05-15 23:13:53 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-05-15 23:13:53 INFO (MainThread) [__main__] Running Supervisor
22-05-15 23:13:53 WARNING (MainThread) [supervisor.core] System running in a unsupported environment!
22-05-15 23:13:53 INFO (MainThread) [supervisor.jobs] 'OSManager.mark_healthy' blocked from execution, no Home Assistant OS available
22-05-15 23:13:53 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.INITIALIZE' starting 0 add-ons
22-05-15 23:13:53 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.SYSTEM' starting 0 add-ons
22-05-15 23:13:53 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.SERVICES' starting 0 add-ons
22-05-15 23:13:53 INFO (MainThread) [supervisor.core] Start Home Assistant Core
22-05-15 23:13:53 INFO (SyncWorker_0) [supervisor.docker.interface] Starting homeassistant
22-05-15 23:13:54 ERROR (SyncWorker_0) [supervisor.docker.interface] Can't start homeassistant: 400 Client Error for http+docker://localhost/v1.41/containers/8f2931dab12408519396a19c66a015187cfd35cfcfbdd211dd03e412bbf83954/start: Bad Request ("failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/home/dietpi/docker_data/hassio/tmp/homeassistant_pulse" to rootfs at "/etc/pulse/client.conf": mount /home/dietpi/docker_data/hassio/tmp/homeassistant_pulse:/etc/pulse/client.conf (via /proc/self/fd/6), flags: 0x5001: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type")
22-05-15 23:13:54 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.APPLICATION' starting 0 add-ons
22-05-15 23:13:54 INFO (MainThread) [supervisor.misc.tasks] All core tasks are scheduled
22-05-15 23:13:54 INFO (MainThread) [supervisor.core] Supervisor is up and running
22-05-15 23:13:54 INFO (MainThread) [supervisor.host.info] Updating local host information
22-05-15 23:13:54 INFO (MainThread) [supervisor.host.manager] Host information reload completed
22-05-15 23:13:54 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
22-05-15 23:13:54 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
22-05-15 23:13:54 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
22-05-15 23:13:55 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
22-05-15 23:13:55 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.CREATE_FULL_BACKUP - ContextType.SYSTEM / None
22-05-15 23:13:55 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_FAILED/ContextType.DNS_SERVER
22-05-15 23:13:55 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
22-05-15 23:13:55 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_IPV6_ERROR/ContextType.DNS_SERVER
22-05-15 23:13:55 INFO (MainThread) [supervisor.resolution.check] System checks complete
22-05-15 23:13:55 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-05-15 23:13:55 WARNING (MainThread) [supervisor.resolution.evaluations.base] Found unsupported images: {'portainer/portainer-ce'} (more-info: https://www.home-assistant.io/more-info/unsupported/software)
22-05-15 23:13:57 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-05-15 23:13:57 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
22-05-15 23:13:57 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete`


i had home assistant working at one point, so i just need to figure out what the issue is

**Comment by pbanj** [comment]
found and fixed the issue
![image](https://user-images.githubusercontent.com/17306233/168517660-889afcf7-7dc7-43cb-a3e4-317c3c37ef49.png)
![image](https://user-images.githubusercontent.com/17306233/168517702-2d9ed3bc-150a-4b03-9b56-5a2d4a47fce2.png)

**Comment by HyperCriSiS** [comment]
Amazing!! Will other services run along with it or will docker eat up all resources?

**Comment by pbanj** [comment]
> Amazing!! Will other services run along with it or will docker eat up all resources? Are there many changes that need to be done to make it work?

that would more depend on the amount of ram you have and how much each thing uses. im using it on a 2gb pi4 and it has enough ram for other stuff to be run(it's what i originally used this pi for way back when i got it). as for changes, not really. mostly it was just taking settings from my working setup on my 8gb pi4 and using them for this test, reading the old install script to figure stuff out, grabbing a few bin files off the 8gb. theres one issue im trying to work out, dbus is messed up and cant figure out why. 

if i knew more about how to make docker compose files i'd make one(prob gonna look into it). but i will upload the system image once im done so it can be used/help with getting it actually added to diet pi

**Comment by pbanj** [comment]
hmmm my install that ive been using for a long time now the supervisor decided it no longer wants to run. the best part is it isnt giving me logs. the test pi is doing the same thing. so thats fun. ill keep things updated.

also a thought occurred to me. with me running everything outside of home assistant, why am i using it supervised. the only addon i ever found useful was the file browser and that is easy to replace by just pointing docker to use an easy to find folder which i already do.

**Comment by pbanj** [comment]
```
version: "3"
services:
  hassio_audio:
    cap_add:
      - SYS_NICE
      - SYS_RESOURCE
    container_name: hassio_audio
    dns:
      - 172.30.32.3
    dns_search:
      - local.hass.io
    domainname: local.hass.io
    entrypoint:
      - /init
    environment:
      - TZ=America/New_York
      - SUPERVISOR_MACHINE=raspberrypi4-64
      - PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
      - LANG=C.UTF-8
      - S6_BEHAVIOUR_IF_STAGE2_FAILS=2
      - S6_CMD_WAIT_FOR_SERVICES=1
    hostname: hassio-audio
    image: ghcr.io/home-assistant/aarch64-hassio-audio:2022.05.0
    ipc: private
    labels:
      io.hass.arch: aarch64
      io.hass.base.arch: aarch64
      io.hass.base.image: arm64v8/alpine:3.13
      io.hass.base.name: alpine
      io.hass.base.version: 2022.02.0
      io.hass.type: audio
      io.hass.version: 2022.05.0
      org.opencontainers.image.authors: The Home Assistant Authors
      org.opencontainers.image.created: '''2022-05-04 15:24:30+00:00'''
      org.opencontainers.image.description: Home Assistant Supervisor plugin for Audio
      org.opencontainers.image.documentation: https://www.home-assistant.io/docs/
      org.opencontainers.image.licenses: Apache License 2.0
      org.opencontainers.image.source: https://github.com/home-assistant/plugin-audio
      org.opencontainers.image.title: Home Assistant Audio Plugin
      org.opencontainers.image.url: https://www.home-assistant.io/
      org.opencontainers.image.version: 2022.05.0
    logging:
      driver: journald
      options: {}
    networks:
      - hassio
    restart: always
    security_opt:
      - seccomp=unconfined
    ulimits:
      - Hard: 10
        Name: rtprio
        Soft: 10
    volumes:
      - /dev:/dev:ro
      - /home/dietpi/docker_data/hassio/audio:/data:rw
      - /run/dbus:/run/dbus:ro
      - /run/udev:/run/udev:ro
      - /etc/machine-id:/etc/machine-id:ro
    working_dir: /
  hassio_cli:
    container_name: hassio_cli
    dns:
      - 172.30.32.3
    dns_search:
      - local.hass.io
    domainname: local.hass.io
    entrypoint:
      - /init
    environment:
      - TZ=America/New_York
      - SUPERVISOR_TOKEN=a3fa37d5e526cc724031159cba129e7b18731ed374543b748ecf695643ee4cd790aeba1dce62ca5d95b0950dbfc14c8f1c24522cbc04b516
      - PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
      - LANG=C.UTF-8
      - S6_BEHAVIOUR_IF_STAGE2_FAILS=2
      - S6_CMD_WAIT_FOR_SERVICES=1
    extra_hosts:
      - observer:172.30.32.6
      - supervisor:172.30.32.2
    hostname: hassio-cli
    image: ghcr.io/home-assistant/aarch64-hassio-cli:2022.05.0
    ipc: private
    labels:
      io.hass.arch: aarch64
      io.hass.base.arch: aarch64
      io.hass.base.image: arm64v8/alpine:3.14
      io.hass.base.name: alpine
      io.hass.base.version: 2022.02.0
      io.hass.type: cli
      io.hass.version: 2022.05.0
      org.opencontainers.image.authors: The Home Assistant Authors
      org.opencontainers.image.created: '''2022-05-02 12:05:24+00:00'''
      org.opencontainers.image.description: Home Assistant Supervisor plugin for CLI
      org.opencontainers.image.documentation: https://www.home-assistant.io/docs/
      org.opencontainers.image.licenses: Apache License 2.0
      org.opencontainers.image.source: https://github.com/home-assistant/plugin-cli
      org.opencontainers.image.title: Home Assistant CLI Plugin
      org.opencontainers.image.url: https://www.home-assistant.io/
      org.opencontainers.image.version: 2022.05.0
    logging:
      driver: journald
      options: {}
    networks:
      - hassio
    restart: always
    security_opt:
      - seccomp=unconfined
    working_dir: /
  hassio_dns:
    container_name: hassio_dns
    entrypoint:
      - /init
    environment:
      - TZ=America/New_York
      - PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
      - LANG=C.UTF-8
      - S6_BEHAVIOUR_IF_STAGE2_FAILS=2
      - S6_CMD_WAIT_FOR_SERVICES=1
    hostname: hassio-dns
    image: ghcr.io/home-assistant/aarch64-hassio-dns:2022.04.1
    ipc: private
    labels:
      io.hass.arch: aarch64
      io.hass.base.arch: aarch64
      io.hass.base.image: arm64v8/alpine:3.14
      io.hass.base.name: alpine
      io.hass.base.version: 2022.02.0
      io.hass.type: dns
      io.hass.version: 2022.04.1
      org.opencontainers.image.authors: The Home Assistant Authors
      org.opencontainers.image.created: '''2022-04-26 14:36:58+00:00'''
      org.opencontainers.image.description: Home Assistant Supervisor plugin for DNS
      org.opencontainers.image.documentation: https://www.home-assistant.io/docs/
      org.opencontainers.image.licenses: Apache License 2.0
      org.opencontainers.image.source: https://github.com/home-assistant/plugin-dns
      org.opencontainers.image.title: Home Assistant DNS Plugin
      org.opencontainers.image.url: https://www.home-assistant.io/
      org.opencontainers.image.version: 2022.04.1
    logging:
      driver: journald
      options: {}
    networks:
      - hassio
    restart: always
    security_opt:
      - seccomp=unconfined
    volumes:
      - /home/dietpi/docker_data/hassio/dns:/config:rw
      - /run/dbus:/run/dbus:ro
    working_dir: /config
  hassio_multicast:
    cap_add:
      - NET_ADMIN
    container_name: hassio_multicast
    dns:
      - 172.30.32.3
    dns_search:
      - local.hass.io
    domainname: local.hass.io
    entrypoint:
      - /init
    environment:
      - TZ=America/New_York
      - PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
      - LANG=C.UTF-8
      - S6_BEHAVIOUR_IF_STAGE2_FAILS=2
      - S6_CMD_WAIT_FOR_SERVICES=1
    extra_hosts:
      - supervisor:172.30.32.2
    hostname: hassio-multicast
    image: ghcr.io/home-assistant/aarch64-hassio-multicast:2022.02.0
    ipc: private
    labels:
      io.hass.arch: aarch64
      io.hass.base.arch: aarch64
      io.hass.base.image: arm64v8/alpine:3.14
      io.hass.base.name: alpine
      io.hass.base.version: 2022.02.0
      io.hass.type: multicast
      io.hass.version: 2022.02.0
      org.opencontainers.image.created: '''2022-02-23 13:03:39+00:00'''
      org.opencontainers.image.source: https://github.com/home-assistant/plugin-multicast
      org.opencontainers.image.version: 2022.02.0
    logging:
      driver: journald
      options: {}
    networks:
      - host
    restart: always
    security_opt:
      - seccomp=unconfined
    working_dir: /
  hassio_observer:
    container_name: hassio_observer
    dns:
      - 172.30.32.3
    dns_search:
      - local.hass.io
    domainname: local.hass.io
    entrypoint:
      - /init
    environment:
      - TZ=America/New_York
      - SUPERVISOR_TOKEN=c0659d8647e1963884c1c689548a39aa568a29f80ed252731ffefd4704eb97a0675e8a21d00b3a082ffb4a05c960e07ad3ca8e2153e07db1
      - NETWORK_MASK=172.30.32.0/23
      - PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
      - LANG=C.UTF-8
      - S6_BEHAVIOUR_IF_STAGE2_FAILS=2
      - S6_CMD_WAIT_FOR_SERVICES=1
    extra_hosts:
      - supervisor:172.30.32.2
    hostname: hassio-observer
    image: ghcr.io/home-assistant/aarch64-hassio-observer:2021.10.0
    ipc: private
    labels:
      io.hass.arch: aarch64
      io.hass.base.arch: aarch64
      io.hass.base.image: arm64v8/alpine:3.14
      io.hass.base.name: alpine
      io.hass.base.version: 2021.09.1
      io.hass.type: observer
      io.hass.version: 2021.10.0
      org.opencontainers.image.authors: The Home Assistant Authors
      org.opencontainers.image.created: '''2021-10-18 09:04:14+00:00'''
      org.opencontainers.image.description: Home Assistant Supervisor plugin monitor
        Supervisor
      org.opencontainers.image.documentation: https://www.home-assistant.io/docs/
      org.opencontainers.image.licenses: Apache License 2.0
      org.opencontainers.image.source: https://github.com/home-assistant/plugin-observer
      org.opencontainers.image.title: Home Assistant Observer Plugin
      org.opencontainers.image.url: https://www.home-assistant.io/
      org.opencontainers.image.version: 2021.10.0
    logging:
      driver: journald
      options: {}
    networks:
      - hassio
    ports:
      - 4357:80/tcp
    restart: always
    security_opt:
      - seccomp=unconfined
    volumes:
      - /run/docker.sock:/run/docker.sock:ro
    working_dir: /
  hassio_supervisor:
    cap_add:
      - AUDIT_WRITE
      - CHOWN
      - DAC_OVERRIDE
      - FOWNER
      - FSETID
      - KILL
      - MKNOD
      - NET_BIND_SERVICE
      - NET_RAW
      - SETFCAP
      - SETGID
      - SETPCAP
      - SETUID
      - SYS_CHROOT
    cap_drop:
      - AUDIT_CONTROL
      - BLOCK_SUSPEND
      - DAC_READ_SEARCH
      - IPC_LOCK
      - IPC_OWNER
      - LEASE
      - LINUX_IMMUTABLE
      - MAC_ADMIN
      - MAC_OVERRIDE
      - NET_ADMIN
      - NET_BROADCAST
      - SYSLOG
      - SYS_ADMIN
      - SYS_BOOT
      - SYS_MODULE
      - SYS_NICE
      - SYS_PACCT
      - SYS_PTRACE
      - SYS_RAWIO
      - SYS_RESOURCE
      - SYS_TIME
      - SYS_TTY_CONFIG
      - WAKE_ALARM
    container_name: hassio_supervisor
    entrypoint:
      - /init
    environment:
      - LANG=C.UTF-8
      - PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
      - S6_BEHAVIOUR_IF_STAGE2_FAILS=2
      - S6_CMD_WAIT_FOR_SERVICES=1
      - S6_SERVICES_GRACETIME=10000
      - SUPERVISOR_API=http://localhost
      - SUPERVISOR_MACHINE=raspberrypi4-64
      - SUPERVISOR_NAME=hassio_supervisor
      - SUPERVISOR_SHARE=/home/dietpi/docker_data/hassio
      - S6_CMD_WAIT_FOR_SERVICES_MAXTIME=0
    hostname: 8cc241af29cc
    image: ghcr.io/home-assistant/aarch64-hassio-supervisor:2022.05.3
    ipc: private
    labels:
      io.hass.arch: aarch64
      io.hass.base.arch: aarch64
      io.hass.base.image: homeassistant/aarch64-base:3.14
      io.hass.base.name: python
      io.hass.base.version: 2022.02.0
      io.hass.type: supervisor
      io.hass.version: 2022.05.0
      org.opencontainers.image.authors: The Home Assistant Authors
      org.opencontainers.image.created: '''2022-05-02 12:12:17+00:00'''
      org.opencontainers.image.description: Container-based system for managing Home
        Assistant Core installation
      org.opencontainers.image.documentation: https://www.home-assistant.io/docs/
      org.opencontainers.image.licenses: Apache License 2.0
      org.opencontainers.image.source: https://github.com/home-assistant/supervisor
      org.opencontainers.image.title: Home Assistant Supervisor
      org.opencontainers.image.url: https://www.home-assistant.io/
      org.opencontainers.image.version: 2022.05.0
    logging:
      driver: journald
      options: {}
    networks:
      - hassio
    privileged: true
    restart: always
    security_opt:
      - apparmor=hassio-supervisor
      - seccomp=unconfined
      - label=disable
    volumes:
      - /run/docker.sock:/run/docker.sock
      - /run/udev:/run/udev:ro
      - /home/dietpi/docker_data/hassio:/data
      - /etc/machine-id:/etc/machine-id:ro
      - /run/dbus:/run/dbus:ro
    working_dir: /
  homeassistant:
    container_name: homeassistant
    dns:
      - 172.30.32.3
    dns_search:
      - local.hass.io
    domainname: local.hass.io
    entrypoint:
      - /init
    environment:
      - HASSIO=172.30.32.2
      - SUPERVISOR=172.30.32.2
      - TZ=America/New_York
      - SUPERVISOR_TOKEN=85c80406681a5f650203ff23b2dc7c7c9fa3b19f2788e1059db85f119f348104cec7fb4965422389b06d8a344c427ada6e14048595df8d99
      - HASSIO_TOKEN=85c80406681a5f650203ff23b2dc7c7c9fa3b19f2788e1059db85f119f348104cec7fb4965422389b06d8a344c427ada6e14048595df8d99
      - PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
      - LANG=C.UTF-8
      - S6_BEHAVIOUR_IF_STAGE2_FAILS=2
      - S6_CMD_WAIT_FOR_SERVICES_MAXTIME=0
      - S6_CMD_WAIT_FOR_SERVICES=1
      - WHEELS_LINKS=https://wheels.home-assistant.io/alpine-3.14/aarch64/
      - S6_SERVICES_GRACETIME=220000
    extra_hosts:
      - observer:172.30.32.6
      - supervisor:172.30.32.2
    hostname: homeassistant
    image: ghcr.io/home-assistant/raspberrypi4-64-homeassistant:2022.6.3
    ipc: private
    labels:
      io.hass.arch: aarch64
      io.hass.base.arch: aarch64
      io.hass.base.image: homeassistant/aarch64-base:3.14
      io.hass.base.name: python
      io.hass.base.version: 2022.05.1
      io.hass.machine: raspberrypi4-64
      io.hass.type: core
      io.hass.version: 2022.6.3
      org.opencontainers.image.authors: The Home Assistant Authors
      org.opencontainers.image.created: '''2022-06-06 23:21:56+00:00'''
      org.opencontainers.image.description: Open-source home automation platform running
        on Python 3
      org.opencontainers.image.documentation: https://www.home-assistant.io/docs/
      org.opencontainers.image.licenses: Apache License 2.0
      org.opencontainers.image.source: https://github.com/home-assistant/core
      org.opencontainers.image.title: Home Assistant
      org.opencontainers.image.url: https://www.home-assistant.io/
      org.opencontainers.image.version: 2022.6.3
    logging:
      driver: journald
      options: {}
    networks:
      - host
    privileged: true
    security_opt:
      - seccomp=unconfined
      - label=disable
    volumes:
      - /dev:/dev:ro
      - /run/dbus:/run/dbus:ro
      - /run/udev:/run/udev:ro
      - /home/dietpi/docker_data/hassio/homeassistant:/config:rw
      - /home/dietpi/docker_data/hassio/ssl:/ssl:ro
      - /home/dietpi/docker_data/hassio/share:/share:rw
      - /home/dietpi/docker_data/hassio/media:/media:rw
      - /etc/machine-id:/etc/machine-id:ro
      - /home/dietpi/docker_data/hassio/tmp/homeassistant_pulse:/etc/pulse/client.conf:ro
      - /home/dietpi/docker_data/hassio/audio/external:/run/audio:ro
      - /home/dietpi/docker_data/hassio/audio/asound:/etc/asound.conf:ro
    working_dir: /config
networks:
  hassio:
    external: true
    name: hassio
  host:
    external: true
    name: host

```

found a tool to create a compose file. also my supervisor decided to just start working again so who knows what the hell is up with that. i still have to work out the dbus issue iirc. i accidently wiped the sd card i was using to test this so i have to start over. but this compose file will make things a lot easier to restart

**Comment by HyperCriSiS** [comment]
> found a tool to create a compose file. also my supervisor decided to just start working again so who knows what the hell is up with that. i still have to work out the dbus issue iirc. i accidently wiped the sd card i was using to test this so i have to start over. but this compose file will make things a lot easier to restart
> Config file attached. Click to expand!

Hi,
How is your supervisor instance running? :)

**Comment by HyperCriSiS** [comment]
> 

You have much more devices to add and also better possibilities to customize the Dashboard which per default is very restricted and kinda bad.

**Comment by pbanj** [comment]
it's still up and running. it has stopped like once or twice after i made that post and has been fine since so not sure what it's issue was

**Comment by pbanj** [comment]
> > 
> 
> You have much more devices to add and also better possibilities to customize the Dashboard which per default is very restricted and kinda bad.

you can add devices and dashboards using hacs and you can install hacs without the supervisor. at this point i feel the supervisor is more of a hindrance than anything. all the addons are just docker containers, so it isnt really any different than just running those on their own. the only difference is that using it as an "addon" in home assistant means if home assistant breaks/wont start for whatever reason, none of the addons will either. which is pretty shitty tbh. it's a lot better to just use portainer if you want a ui to manage docker containers.

**Comment by MichaIng** [comment]
> you can install hacs without the supervisor

Interesting: https://www.wundertech.net/how-to-install-hacs-on-home-assistant/

Probably we can do this even with the native HA installation. Docker is still required for the add-ons, but in theory?

**Comment by MichaIng** [comment]
Ah, I understand just now that HACS is not for addons, but for additional integrations, themes and other customisation, isn't it? Thanks for the link, so it can be installed into HA core as well. Looks like a reasonable addition to our HA implementation, but not related to HA supervised.

**Comment by pbanj** [comment]
> Ah, I understand just now that HACS is not for addons, but for additional integrations, themes and other customisation, isn't it? Thanks for the link, so it can be installed into HA core as well. Looks like a reasonable addition to our HA implementation, but not related to HA supervised.

correct, it might be enough to make people not want to bother with supervised.

**Comment by HyperCriSiS** [comment]
> > Ah, I understand just now that HACS is not for addons, but for additional integrations, themes and other customisation, isn't it? Thanks for the link, so it can be installed into HA core as well. Looks like a reasonable addition to our HA implementation, but not related to HA supervised.
> 
> correct, it might be enough to make people not want to bother with supervised.

But without the supervisor addons are not possible, or is there also a way without supervisor?

**Comment by MichaIng** [comment]
At least you'd need to manage the addon's containers manually. So yeah, it is not exactly user friendly to use addons without HA supervised.

**Comment by pbanj** [comment]
Could use portainer. Then it's pretty much the same experience. A better one when you realize you can set the data locations to whatever you want so you can easily find them instead of wherever HA puts it

**Comment by MichaIng** [comment]
That's true. But I guess users would like to have all HA related stuff in a single web UI and not a second process/webserver/UI for the addons. Also it is simpler to find new addons via HA supervised, compared to browsing the Docker registry or some website manually for them.

But it's good that this is an option as well for those who do not have an issue with above points.

**Comment by HyperCriSiS** [comment]
Is there any chance that the supervisor could be part of the next releases? :grinning:

**Comment by Joulinar** [comment]
I don't think so.

**Comment by MichaIng** [comment]
More likely HACS, but without the supervisor.

**Comment by MichaIng** [comment]
Shall we close this issue with the merge of #5802 (HACS integration)? I don't think the whole HA Supervised is feasible for DietPi integration in general.

**Comment by Joulinar** [comment]
I agree with you @MichaIng 
That's the easiest we can do.

**Comment by MichaIng** [comment]
Merged
